cli/sawtooth_cli/main.py

```diff
@@ -83,6 +84,18 @@ def create_parent_parser(prog_name):
         action='count',
         help='enable more verbose output')
 
+    try:
+        version = pkg_resources.get_distribution('sawtooth-cli').version
+    except pkg_resources.DistributionNotFound:
+        print("please install sawtooth-cli with setup.py")
+        return
+
+    parent_parser.add_argument(
+        '-V', '--version',
+        action='version',
```

Comment:
I think you want action='store_true' as the -V flag is a boolean. https://docs.python.org/3.5/howto/argparse.html.